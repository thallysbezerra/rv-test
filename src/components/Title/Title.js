import React from 'react';
import PropTypes from 'prop-types';

import './Title.scss';

const Title = ({ children, className }) => {
	return <h2 className={`title ${className}`}>{children}</h2>;
};

Title.propTypes = {
	children: PropTypes.any.isRequired,
	className: PropTypes.string.isRequired
};

Title.defaultProps = {
	children: "Title",
	className: ""
}

export default Title;
