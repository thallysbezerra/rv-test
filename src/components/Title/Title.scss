@import '../../css/theme.scss';

.title {
	font-size: 22px;
	font-weight: $light;
	margin: 0 0 40px;

	@include breakpoint(tabletL) {
		font-size: 45px;
	}
}
