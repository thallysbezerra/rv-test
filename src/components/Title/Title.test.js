import React from 'react';
import Title from './Title';

import renderer from 'react-test-renderer';

describe('Title UI', () => {
	it('<Title /> must have the snapshot version.', () => {
		const tree = renderer.create(<Title />);

		expect(tree.toJSON()).toMatchSnapshot();
	});
});
