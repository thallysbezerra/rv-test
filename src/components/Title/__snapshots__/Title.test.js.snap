// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Title UI <Title /> must have the snapshot version. 1`] = `
<h2
  className="title "
>
  Title
</h2>
`;
